UnoCheck/Checkups/XCodeCheckup.cs

```diff
@@ -198,6 +216,21 @@ XCodeInfo GetXcodeInfo(string path, bool selected)
 			}
 			return null;
 		}
+
+		static bool ValidateForiOSSDK(XCodeInfo selected)
+		{
+			var iphoneSimulatorSDKPath = Path.Combine(selected.Path, "Contents", "Developer", "Platforms", "iPhoneSimulator.platform", "Developer", "SDKs", "iPhoneSimulator.sdk");
+
+			if (Directory.Exists(iphoneSimulatorSDKPath))
+			{
+				return true;
```

Comment:
If you delete the iOS SDK (from Xcode Component's window) then the directory will still be present, so the check will pass but the SDK will be missing (at build time).